packages/sensor/lib/sensor.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+const EventChannel _accelerometerEventChannel =
+    const EventChannel('plugins.flutter.io/accelerometer');
+
+const EventChannel _gyroscopeEventChannel =
+    const EventChannel('plugins.flutter.io/gyroscope');
+
+Stream<List<double>> _accelerometerEvents;
+Stream<List<double>> _gyroscopeEvents;
+
+List<double> numToDouble(List<num> l) {
+  return l.map((num v) => v.toDouble());
+}
+
+/// A broadcast stream of events from the device accelerometer.
```

Comment:
In fact, it may be better to replace `List<double>` with more expressive per-sensor classes like `AccelerometerEvent` and `GyroscopeEvent`. These would both contain three doubles (x,y,z) though with different meanings. The event classes could also contain timestamp and accuracy, if we want to add that at some point (probably depends on what iOS has to offer here). A main point of introducing per-sensor classes is that they would provide a perfect place to put documentation of what x, y, and z mean in each case.